components/automate-cli/pkg/verifyserver/services/batchcheckservice/batchcheckservice.go

```diff
@@ -135,15 +139,21 @@ func constructBatchCheckResponse(checkTriggerRespMap map[string][]models.CheckTr
 	var result = make([]models.BatchCheckResult, len(ipMap))
 	var resultIndex = 0
 	for _, v := range ipMap {
+		if len(v) == 0 {
+			continue
+		}
 		result[resultIndex].Ip = v[0].Host
 		result[resultIndex].NodeType = v[0].NodeType
 		resultArray := []models.ApiResult{}
 		for _, checkResult := range v {
```

Comment:
Can we get the checktype from the response and append message accordingly ?

Reply:
Yes we are getting the check type in checkResult.CheckType.